app/presenters/courses_presenter.rb

```diff
@@ -22,15 +22,36 @@ def campaign
     @campaign
   end
 
+  def campaigns
+    Campaign.active
+  end
+
   def courses
     campaign.courses
   end
 
+  def active_courses
+    campaign.courses.current_and_future
+  end
+
   def courses_by_recent_edits
     # Sort first by recent edit count, and then by course title
     courses.sort_by { |course| [-course.recent_revision_count, course.title] }
   end
 
+  def active_courses_by_recent_edits
+    active_courses.sort_by { |course| [-course.recent_revision_count, course.title] }
+  end
+
+  def campaigns_by_num_courses
+    # Sort first by number of courses, then by campaign title
+    campaigns.sort_by { |campaign| [-campaign.courses.count, campaign.title] }
+  end
+
+  def can_create?
```

Comment:
Since we're mixing courses and campaigns here, this is probably better named `can_create_campaign` to avoid confusion.